orb/core/column.py

```diff
@@ -484,12 +490,3 @@ def fromJSON(cls, jdata):
             col = col_cls()
             col.loadJSON(jdata)
             return col
-
-
-class VirtualColumn(Column):
```

Comment:
removing virtualization this way in favor of more specific approach:

    @orb.virtual(BooleanColumn)
    def isEnabled(self):
        return False